tools/eslint/package.json

```diff
@@ -14,19 +14,19 @@
     ]
   ],
   "_from": "eslint@latest",
-  "_id": "eslint@3.19.0",
+  "_id": "eslint@4.0.0",
```

Comment:
Yes "build" from git, or download `https://registry.npmjs.org/eslint/-/eslint-4.0.0.tgz` than `npm i` dependencies.
I have no idea what could be the difference, mostly wondering.